.konflux/container-build.args

```diff
@@ -0,0 +1,2 @@
+OPERATOR_VERSION=4.20.0
```

Comment:
why do we need a zstream specificed?

Reply:
to populate the docker labels:https://github.com/openshift-kni/numaresources-operator/blob/57acc8cb8e2a764a72f26f0251d11a4ba6f3a92d/.konflux/operator/konflux.Dockerfile#L32 

We used to do that with z streams so far so this is a placeholder at the moment in case we introduce a skip mechanism . the end result is the same as hardcoding this value

Reply:
cc @rauhersu @fontivan  I assume you will also have to deal with semver label bumps when it comes to z stream bumps